intermediate_source/torch_export_tutorial.py

```diff
@@ -0,0 +1,462 @@
+# -*- coding: utf-8 -*-
+
+"""
+torch.export Tutorial
+================
+**Author:** William Wen, Zhengxu Chen
+"""
+
+######################################################################
+# ``torch.export`` is the PyTorch 2.0 way to export PyTorch models intended
+# to be run on high performance environments.
```

Comment:
Its jut a way of getting a full graph out of Pt2, no more or less. There's nothing high performance env opinionated about it. I would make this an example of a point on the former.